packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js

```diff
@@ -232,21 +228,35 @@ export default class QuanticStandaloneSearchBox extends NavigationMixin(
 
   clearInput() {
     this.input.value = '';
-    this.actualSearchBox.updateText(this.input.value);
+    this.standaloneSearchBox.updateText(this.input.value);
     this.input.focus();
   }
 
   handleSuggestionSelection(event) {
     const textValue = event.detail;
-    this.actualSearchBox.selectSuggestion(textValue);
+    this.standaloneSearchBox.selectSuggestion(textValue);
+  }
+
+  resetSearchboxState(engineId) {
+    const engine = getHeadlessBindings(engineId)?.engine;
+    if(!engine) {
+      return;
+    }
+    const {updateQuery} = CoveoHeadless.loadQueryActions(engine);
+
+    engine.dispatch(updateQuery({q: ''}));
   }
 
   navigateToSearchPage() {
+    const value = this.standaloneSearchBox.state.value;
+
+    this.resetSearchboxState(this.engineId);
```

Comment:
Why do you need to reset the searchbox state if you delete the engine whenever you navigate away from the search page?

This seems to me like this will always go through Line#242 no?
Or at least that it should, if it doesn't it would mean there's a flaw in the logic to delete the engine no?

Reply:
yeah you're right that i don't need to reset the regular searchbox state anymore